marklogic-data-hub-central/ui/e2e/cypress/e2e/mastering/e2eMasteringflow.cy.tsx

```diff
@@ -427,52 +332,89 @@ describe("Validate E2E Mastering Flow", () => {
+    cy.waitForAsyncRequest();
+
+    browsePage.getFacetItemCheckbox("collection", "sm-Patient-merged").click();
     entitiesSidebar.applyFacets();
     browsePage.waitForSpinnerToDisappear();
     cy.waitForAsyncRequest();
-    browsePage.waitForCardToLoad();
-    browsePage.getTotalDocuments().should("eq", 12);
-    browsePage.getFacetItemCheckbox("collection", "sm-Patient-mastered").click();
+    browsePage.getTotalDocuments().should("eq", 3);
+    cy.fixture("patients/e2eMasteringflow/sm-patient-merged").then(smPatienMerged => {
+      browsePage.hcTableRows.should("have.length", 3).each(($row, index) => {
+        expect($row.text()).to.contain(smPatienMerged[index].firstName);
+        expect($row.text()).to.contain(smPatienMerged[index].lastName);
+        expect($row.text()).to.contain(smPatienMerged[index].ssn);
+        expect($row.text()).to.contain(smPatienMerged[index].zipCode);
+        expect($row.text()).to.contain(smPatienMerged[index].address);
+        expect($row.text()).to.contain(smPatienMerged[index].dateOfBirth);
+      });
+    });
+
+    browsePage.getClearAllFacetsButton().click();
     browsePage.waitForSpinnerToDisappear();
-    browsePage.getFacetItemCheckbox("collection", "sm-Patient-merged").click();
+    cy.waitForAsyncRequest();
+
+    entitiesSidebar.toggleAllDataView();
+
+    cy.log("Reloading available collections");
+    browsePage.showMoreCollection();
+    browsePage.showMoreCollection();
+
+    browsePage.getFacetItemCheckbox("collection", "sm-Patient-archived").click();
     entitiesSidebar.applyFacets();
     browsePage.waitForSpinnerToDisappear();
     cy.waitForAsyncRequest();
     browsePage.waitForCardToLoad();
-    browsePage.getTotalDocuments().should("eq", 2);
-    browsePage.getFacetItemCheckbox("collection", "sm-Patient-merged").click();
-    browsePage.waitForSpinnerToDisappear();
+    browsePage.getTotalDocuments().should("eq", 6);
+    smPatientArchivedCollection.forEach((cardName: string) => {
+      browsePage.verifyCardExist(cardName);
+    });
+
+    browsePage.getFacetItemCheckbox("collection", "sm-Patient-archived").click();
+    cy.waitForAsyncRequest();
     browsePage.getFacetItemCheckbox("collection", "sm-Patient-auditing").click();
     entitiesSidebar.applyFacets();
     browsePage.waitForSpinnerToDisappear();
     cy.waitForAsyncRequest();
     browsePage.waitForCardToLoad();
-    browsePage.getTotalDocuments().should("eq", 2);
+    browsePage.getTotalDocuments().should("eq", 3);
+
     browsePage.getFacetItemCheckbox("collection", "sm-Patient-auditing").click();
-    browsePage.waitForSpinnerToDisappear();
+    cy.waitForAsyncRequest();
     browsePage.getFacetItemCheckbox("collection", "sm-Patient-notification").click();
     entitiesSidebar.applyFacets();
     browsePage.waitForSpinnerToDisappear();
     cy.waitForAsyncRequest();
     browsePage.waitForCardToLoad();
-    browsePage.getTotalDocuments().should("eq", 6);
```

Comment:
@vmendez-marklogic Are you planning to add validations similar to sm-Patient-mastered and sm-Patient-merged for sm-Patient-archived, sm-Patient-auditing, sm-Patient-notification?

Reply:
https://project.marklogic.com/jira/browse/DHFPROD-10438 => As we discuss, let investigate how to validate the auditing nad notification collection. We can merge this task